lib/sda/service_test.go

```diff
@@ -444,6 +449,34 @@ func TestServiceBackForthProtocol(t *testing.T) {
 	assert.Equal(t, resp.Val, 10)
 }
 
+func TestClient_Send(t *testing.T) {
```

Comment:
Hi, I won't vote if there should be an _ or not here (just because I don't find it very important) ... But: @ineiti the link you provided does not say it is convention to put a `_` in between (only for examples: `Example_ForBlaBlaEtc`). In the google code itself you won't find an underscore (only in the Example tests). But I'm OK to make this a convention in *our* code (as it's not wrong/not a problem to put an `_` in between Struct and member function).